examples/basics/Cargo.toml

```diff
@@ -1,8 +1,9 @@
 [package]
 name = "examples-basics"
-version = "0.1.0"
+version = "0.0.0"
```

Comment:
Do we also want the explanation?
https://github.com/heroku/procfile-cnb/pull/38/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R3

(Happy either way)

Reply:
I don't think we really need it. I'll go ahead and merge, we could add it later if we want to.